suite-native/module-settings/src/components/ApplicationSettings.tsx

```diff
@@ -9,12 +11,30 @@ import {
     RootStackParamList,
     RootStackRoutes,
 } from '@suite-native/navigation';
+import { Text } from '@suite-native/atoms';
 import { isDevelopOrDebugEnv } from '@suite-native/config';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 
 import { SettingsSection } from './SettingsSection';
 import { SettingsSectionItem } from './SettingsSectionItem';
 
+const devButtonStyle = prepareNativeStyle(() => ({
+    position: 'absolute',
+    zIndex: 1,
+    top: 0,
+    right: 0,
+    width: 50,
+    height: 50,
+}));
+
+const isDevButtonVisibleAtom = atom<boolean>(false);
```

Comment:
I think we should use storage atom to persist value

Reply:
Do we won't to store it? It would be same as activating developer options on chrome or iPhone. I'm fine with that, but isn't this just going to pollute the app unnecessarily after resolving users issue? Maybe let's open this topic during standup with @mnuky as well.

Reply:
Yes that my point, it would be same as all other apps.

Reply:
Persisted atom.
https://github.com/trezor/trezor-suite/pull/7962/commits/564357fd962741e9197a6de461ee8b7fb9d41960